openmrs-19/src/test/java/org/motechproject/openmrs19/tasks/OpenMRSTaskDataProviderTest.java

```diff
@@ -264,4 +276,48 @@ public void shouldReturnProviderForLookupGetProviderByUuid() {
         assertEquals(provider, object);
         verify(providerService).getProviderByUuid(null, "5");
     }
+
+    @Test
+    public void shouldReturnRelationshipForLookupGetRelationshipByTypeUuidAndPersonBUuid() {
```

Comment:
Whole OpenMRSTaskDataProvider could use some integration tests. I'll create a ticket for it.